<!doctype html>
<html>

<head>
  <title>Guideline fallback</title>
</head>

<body>
  <div class="test-area">
    <p>This is a paragraph.</p>
    <p class="error">
      This is a paragraph we are testing for.
      <img src="../assets/rex.jpg" />
    </p>
    <h2 class="callback-error">Heading!</h2>

  </div>
  <script src="../quail-testrunner.js"></script>
  <script>
    (function(global) {
      var found = false;
      global.quailTest = function() {
        asyncTest('guidelineFallback', function() {
          // @todo, figure out why this is expect(2). Somewhere, there's another
          // assertion being made. It should only be expect(1);
          expect(1);
          $('.test-area').quail({
            jsonPath : '../../dist',
            reset : true,
            complete : function(eventName, thisTest, _case) {
              if (thisTest.get('name') === 'imgHasAlt' && !found) {
                equal('IMG', _case.get('element').tagName, 'Default guideline has run');
                found = true;
                start();
              }
            }
          });
        });
      };
    }(this));
  </script>
</body>
</html>
